include/io.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+#include <auxiliary.h>
```

Comment:
move the sviper inlcudes below the standard includes

Reply:
Right, I meant to ask. In general, what is your preference for include order?

Standard

SViper

SeqAn

Like that?